client/src/data/testimonials.js

```diff
@@ -1,44 +1,41 @@
 export default [
   {
-    person: 'Kendalyn Hesterson',
-    picture:
-      'https://media.licdn.com/dms/image/C5603AQE4fiq2wYLN4w/' +
-      'profile-displayphoto-shrink_200_200/0?e=1575504000&v=beta&' +
-      't=ENWJca_rj3uyCKSjckj0IkVQj-Z__6hCr9-rSIDvQOc',
-    country: 'Switzerland',
-    testimonial: `Lorem ipsum dolor amet mustache knausgaard +1, 
-            blue bottle waistcoat tbh semiotics artisan synth stumptown 
-            gastropub cornhole celiac swag. Brunch raclette vexillologist 
-            post-ironic glossier ennui XOXO mlkshk godard pour-over blog 
-            tumblr humblebrag. Blue bottle put a bird on it twee prism 
-            biodiesel brooklyn. Blue bottle ennui tbh succulents.`,
+    person: 'Daniella Di Stefano',
+    picture: '/asset/images/daniella_stefano.JPG',
+    // 'profile-displayphoto-shrink_200_200/0?e=1575504000&v=beta&' +
+    // 't=ENWJca_rj3uyCKSjckj0IkVQj-Z__6hCr9-rSIDvQOc',
+    country: 'Legal Services Intern',
```

Comment:
Let's go ahead and change all the `country`s to `title`s to avoid confusion in the future. The only other thing that you should have to change after these three instances are [here](https://github.com/hack4impact-uiuc/ymca/blob/7940918f51c98ca39be46343cb4dc695b2594401/client/src/components/Home.js#L100), where instead of `element.country` you can just put in `element.title`.

Reply:
Okay! Made the changes & cleaned up our homepage :)

Reply:
Looks great - feel free to merge 🐢